src/db/util/raw-model.ts

```diff
@@ -103,6 +111,12 @@ export const defineRawModel =
       return validateAndFilter(res[0]);
     };
 
+    const createMany: CreateManyFn<F> = async (data, opts) => {
+      const builder = opts?.trx ? tbl().transacting(opts.trx) : tbl();
+      const res = await builder.insert(data).returning('*');
+      return validateAndFilter(res[0]);
```

Comment:
I'd probably return all of them rather than just the first one (which may also potentially be null / undefined if a zero-length list is given).